sky/packages/sky/lib/src/material/material_app.dart

```diff
@@ -73,18 +75,24 @@ class _MaterialAppState extends State<MaterialApp> {
   }
 
   Widget build(BuildContext context) {
-    return new Theme(
-      data: config.theme ?? new ThemeData.fallback(),
-      child: new DefaultTextStyle(
-        style: _errorTextStyle,
-        child: new DefaultAssetBundle(
-          bundle: _defaultBundle,
-          child: new Title(
-            title: config.title,
-            child: new Navigator(
-              key: _navigator,
-              routes: config.routes,
-              onGenerateRoute: config.onGenerateRoute
+    return new SizeObserver(
+      onSizeChanged: (Size size) => setState(() { _size = size; }),
```

Comment:
This won't work, it'll mean that the first frame, the MediaQuery will give bogus data.

Do we actually need the size of the app, or is the orientation of the device sufficient? The latter we can get via dart:ui, along with a notification system for handling changes.